DigitalLearningSolutions.Web/Views/LearningMenu/_TutorialCard.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿<div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
+  <div class="nhsuk-card__content">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-three-quarters">
+        <h3 class="nhsuk-card__heading">
+          Introduction to Assessments
+        </h3>
+      </div>
+      <div class="nhsuk-grid-column-one-quarter">
+        <h3 class="nhsuk-heading-xs nhsuk-u-secondary-text-color">
+          Not started
+        </h3>
+      </div>
+    </div>
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <a class="nhsuk-button nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0" asp-action="Tutorial" asp-route-tutorialId="1071" asp-route-sectionId="232" asp-route-customisationId="9865">
```

Comment:
Is this a real action which exists at the moment? If the tutorial page hasn't been created yet then it's better for this button to do nothing for now, and whoever merges last out of you and @benjimarshall can make it use the correct action 🙂

Reply:
The action doesn't exist yet so I'll remove that.